modular_skyrat/modules/milker/code/milker.dm

```diff
@@ -0,0 +1,305 @@
+#define VOLUME_LIGHT 1 // Volume 'light' mode extracts per action
+#define VOLUME_STRONG 5 // Volume 'strong' mode extracts per action
+#define HANDLE_LIGHT 1 // 'light' mode
+#define HANDLE_STRONG 2 // 'strong' mode
+#define HANDLE_MODE_LOWEST 1 // The lowest numbered setting currently available (so we dont need yet another list)
+#define HANDLE_MODE_HIGHEST 2 // The highest numbered setting currently available
+#define PLEASURE_LIGHT 1 // How much pleasure is gained through 'light' mode
+#define PLEASURE_STRONG 3 // How much pleasure is gained through 'strong' mode
+#define AROUSAL_LIGHT 1 // How much arousal is gained through 'light' mode
+#define AROUSAL_STRONG 3 // How much arousal is gained through 'strong' mode
+#define PAIN_LIGHT 3 // How much pain is gained through 'light' mode
+#define PAIN_STRONG 10 // How much pain is gained through 'strong' mode
```

Comment:
Put the comments on the line before and with `///` in front instead, for autodoc stuff.

Reply:
Forgot about that, now its like it should be!